### PR TITLE
Adding divided flag and title color to the collapsible section compon…

### PR DESCRIPTION
**Pull request by avgdorom**
…ent.

### CODE REVIEW

File: projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section-example.module.ts [hunk 1]
```diff
@@ -337,6 +340,7 @@ export class CollapsibleSectionExample2Component implements OnChanges {
   @Input() disabled = false;
 
   @Input() title = mockText(randomNumber(2, 5));
+  @Input() titleColor = '#5555ff';
```
Comment (doroncy) [inline projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section-example.module.ts]: I dont want to have default color like this, please remove this and try to make the ngStyle works when no value in titleColor

File: projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section-example.module.ts [hunk 2]
```diff
@@ -337,6 +340,7 @@ export class CollapsibleSectionExample2Component implements OnChanges {
   @Input() disabled = false;
 
   @Input() title = mockText(randomNumber(2, 5));
```
Comment (doroncy) [inline projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section-example.module.ts]: also remove the mocktext it looks like pasha forgot it here

File: projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section-example.module.ts (hunk 1 above)
Comment (avgdorom) [reply projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section-example.module.ts]: But it's just for the example in Storybook. There's no default in the actual component.

File: projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section-example.module.ts (hunk 2 above)
Comment (avgdorom) [reply projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section-example.module.ts]: What should be here?

**Pull request merged**